dataactcore/models/jobModels.py

```diff
@@ -35,8 +49,10 @@ class Submission(Base):
     datetime_utc = Column(DateTime)
     user_id = Column(Integer, nullable=False) # This refers to the users table in the User DB
     cgac_code = Column(Text)
-    reporting_start_date = Column(Date)
-    reporting_end_date = Column(Date)
+    reporting_start_date = Column(Date, nullable=False)
+    reporting_end_date = Column(Date, nullable=False)
+    reporting_fiscal_year = Column(Integer, nullable=False, default=generateFiscalYear, server_default='0')
```

Comment:
the only reason it had 0 was because we are adding a non-nullable column which for legacy data would have nulls, so it sets the default to 0 to handle those potential errors. This there for a one time solution because the `reporting_end_date` that it needs should never be null, that's why I used server_default vs following what the `concatTas` does, but I can see why that route would be helpful but again, it'll only be need for this initial release of this column